deflect/qt/QmlStreamer.cpp

```diff
@@ -66,6 +66,11 @@ QQmlEngine* QmlStreamer::getQmlEngine()
     return _impl->getQmlEngine();
 }
 
+Stream *deflect::qt::QmlStreamer::getStream()
```

Comment:
The wiring as done there: https://github.com/BlueBrain/DisplayCluster/pull/264/files#diff-759187a34ccce2f9a287e4731e98066dR76 looks reasonable.
The alternative to not exposing the whole Stream object is be to wrap it here in a small class which extends QObject and expose in QML with engine->setContextProperty(...). The class would have just a few selected public methods such as openFile(QString), and any QML streamer could use this method.